src/lib/sdk/Banner.svelte

```diff
@@ -161,6 +163,209 @@
   }
 </script>
 
+<style>
```

Comment:
I think we had this completely separate for full customizablity.

Reply:
Resolved e6b6ed1f6a4baae7e440d22020ce87c995c1a485